codes/rust/chapter_graph/graph_adjacency_list.rs

```diff
@@ -0,0 +1,156 @@
+/*
+ * File: graph_adjacency_list.rs
+ * Created Time: 2023-07-12
+ * Author: night-cruise (2586447362@qq.com)
+ */
+
+use std::collections::HashMap;
+
+/* 顶点类型 */
+#[derive(Copy, Clone, Hash, PartialEq, Eq)]
```

Comment:
Could u plz extract the Vertex class and vals_to_vets vets_to_vals functions to the public module? You can refer to cpp code `codes/cpp/utils/vertex.hpp`